irrd/mirroring/nrtm_generator.py

```diff
@@ -21,6 +22,7 @@ def generate(
         serial_end_requested: Optional[int],
         database_handler: DatabaseHandler,
         remove_auth_hashes=True,
+        dummify_object_text=True,
```

Comment:
This specifically relates to the client IP. The actual decision on alteration is based on more parameters later on. So I think this needs a more narrow name, like (note inverse logic): `client_is_dummifying_exempt=False`.

Reply:
Done, thanks